lua/entities/gmod_wire_expression2/core/glon.lua

```diff
@@ -331,3 +366,166 @@ e2function table vonDecodeTable(string data)
 	local safeTable = sanitizeGlonOutput( self, ret, "t" )
 	return safeTable or table.Copy(DEFAULT)
 end
+
```

Comment:
So we don't have to add new files. Remember, gmod has a shitty as fuck file limit. Might as well place all such encoding functions in the same file. We could, however, rename the file.